intel-retail/automated-vending
# feat(device-mqtt): Migrate device-mqtt service files and add ENV overrides for mqtt topics

closes #123 

## PR Checklist



## What are you changing?


## Issue this PR will close

close: #**123**

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable


Copy the following code block into the `docker-compose.edgex.yml` file
```
  ui:
    container_name: edgex-ui-go
    environment:
      EDGEX_SECURITY_SECRET_STORE: "false"
      SERVICE_HOST: edgex-ui-go
    hostname: edgex-ui-go
    image: nexus3.edgexfoundry.org:10004/edgex-ui:latest
    networks:
      edgex-network: null
    ports:
    - mode: ingress
      target: 4000
      published: "4000"
      protocol: tcp
    read_only: true
    restart: always
    security_opt:
    - no-new-privileges:true
    user: 2002:2001
    volumes:
    - type: bind
      source: /etc/localtime
      target: /etc/localtime
      read_only: true
      bind:
        create_host_path: true
```

- Run `make run-edgex` and view the EdgeX UI and look that the Device Service's profile and device were added correctly
- Check `device-mqtt` container logs and see that files were found and Profile and Device were added properly

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-vending )
